src/benchmarks/micro/coreclr/perflab/LowLevelPerf.cs

```diff
@@ -224,7 +224,25 @@ public void StaticIntPlus()
             for (int i = 0; i < InnerIterationCount; i++)
                 Class.aStaticInt += 1; // TODO: reconsider the design, the cost of inner loop iteration is as big as the benchmarked thing
         }
-        
+
+        [ThreadStatic]
+        private static int t_value = 0;
+
+        [Benchmark]
+        public int ReadThreadStaticValue()
+        {
+            return t_value;
+        }
+
+        [ThreadStatic]
+        private static object t_object = "q";
+
+        [Benchmark]
+        public object ReadThreadStaticObject()
+        {
+            return t_object;
+        }
```

Comment:
We already have:
https://github.com/dotnet/performance/blob/b142ff9c57d3807677bf228635460e40be676591/src/benchmarks/micro/corefx/System.Threading/Perf.ThreadStatic.cs